src/main/target/common_pre.h

```diff
@@ -57,7 +57,7 @@
 #define USE_DSHOT_TELEMETRY_STATS
 #define USE_RPM_FILTER
 #define USE_DYN_IDLE
-#define I2C3_OVERCLOCK true
+#define I2C3_CLOCKSPEED 800
```

Comment:
The existing code is overcooking it a bit here - if we check to ensure `I2C<x>_CLOCKSPEED` is defined in `common_post.h` then there isn't a reason to define it here as well.
And defining it here has the drawback that any custom target that wants to use a different value has to `#undef` first to avoid a warning.

Reply:
I agree. Do you mean ```common_defaults_post.h```? Because that's where we're already doing the checking and defining if not defined

Reply:
Ah yes, we split off the defaults, correct.

Reply:
Done!